<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PureBook-S14" ignored="1" name="Test(failed)" tests="3" failures="1" timestamp="2023-01-19T07:07:22 IST" time="0.059" errors="0">
  <testcase name="password" time="0.011" classname="com.testng.Demo">
    <skipped/>
  </testcase> <!-- password -->
  <testcase name="password" time="0.002" classname="com.testng.Demo">
    <skipped/>
  </testcase> <!-- password -->
  <testcase name="password" time="0.001" classname="com.testng.Demo">
    <failure type="java.lang.AssertionError" message="expected [123] but found [124]">
      <![CDATA[java.lang.AssertionError: expected [123] but found [124]
at com.testng.Demo.password(Demo.java:17)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- password -->
  <testcase name="username1" time="0.0" classname="com.testng.Demo">
    <ignored/>
  </testcase> <!-- username1 -->
</testsuite> <!-- Test(failed) -->
